devito/operator.py

```diff
@@ -430,7 +344,7 @@ def _profile_sections(self, nodes, parameters):
         """Introduce C-level profiling nodes within the Iteration/Expression tree."""
         nodes, profiler = create_profile('timers', nodes)
         self._globals.append(profiler.cdef)
-        parameters.append(Object(profiler.name, profiler.dtype, profiler.new))
+        parameters.append(Object(profiler.name, profiler.dtype, profiler.new()))
```

Comment:
is this unpicking what I did ?

Reply:
Indeed. This will have to go before it is merged. TODO.